app/modules/social_groups/parameters.py

```diff
@@ -88,8 +80,11 @@ def replace(cls, obj, field, value, state):
     def remove(cls, obj, field, value, state):
         # Any researcher can remove a member in the group
         ret_val = False
-        if field == 'member':
-            ret_val = obj.remove_member(value)
-            msg = f'Removing member {value}'
+        if field == 'members':
+            if not isinstance(value, list):
+                value = [value]
```

Comment:
members = [value] 
would be clearer than reusing the value variable, then member rather han single character v

Reply:
Fixed